doc/source/tutorials/configuration/configuration.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: ../../include/global.rst
+
+.. _tutorials-configuration:
+
+======================
+Configuration Instance
+======================
+
+This example shows how to use |igraph|'s `configuration instance <https://igraph.org/python/doc/api/igraph.configuration.Configuration.html>`_ to set default |igraph| settings. This is useful for setting global settings so that they don't need to be explicitly stated at the beginning of every |igraph| project you work on.
+
+First we define the default plotting backend, layout, and color palette, and save them. By default, ``config.save()`` will save files to ``~/.igraphrc`` on Linux and Max OS X systems, or in ``C:\Documents and Settings\username\.igraphrc`` for Windows systems.
+
+.. code-block:: python
+
+    import igraph as ig
+
+    # Get the configuration instance
+    config = ig.Configuration().instance()
```

Comment:
Just use `ig.config`, that's the same instance.

Reply:
Ah that's alot cleaner, thanks!